src/flamenco/runtime/tests/fetch_and_generate.sh

```diff
@@ -0,0 +1,9 @@
+if [ ! -d protosol ]; then
+  git clone --depth=1 https://github.com/firedancer-io/protosol.git
+else
+  cd protosol
+  git pull
+  cd ..
+fi
+
+nanopb_generator.py -I ../.. -I ./protosol/proto -L "" -C ./protosol/proto/*.proto
```

Comment:
Idea: can we have nanopb_generator a part of the 3rd party deps installed in `deps.sh` and have this point to that?

Reply:
lmk if you want me to look into this

Reply:
That wouldn't be a bad idea

Reply:
we should definitely do that, some older versions of nanopb have trouble compiling the newer protobuf format